### PR TITLE
fix some yaml lint attributes

### PR DESCRIPTION
**Pull request by maorfr**

### CODE REVIEW

**Review by patjlm — APPROVED** [review]

File: schemas/cloudflare/terraform-resource-1.yml [hunk 1]
```diff
@@ -156,8 +156,6 @@ properties:
     type: string
   kind:
     type: string
-  name:
-    type: string
```
Comment (patjlm) [inline schemas/cloudflare/terraform-resource-1.yml:160]: why is that removed?

Comment (maorfr) [reply schemas/cloudflare/terraform-resource-1.yml:160]: https://github.com/app-sre/qontract-schemas/pull/523/commits/dfa00c23074eb65c5fdf1450059beac9596a4b30

Comment (patjlm) [reply schemas/cloudflare/terraform-resource-1.yml:160]: Ah I see, duplicate...
